src/website/views/components/toolbar/_examples.pug

```diff
@@ -762,6 +762,406 @@ h2 Examples
+                <div class="chi-form__item -mb--2">
+                  <label class="chi-label" for="example-ba12">Label</label>
+                  <select class="chi-select" id="example-ba12">
+                    <option value="">Select</option>
+                    <option>Option 1</option>
+                    <option>Option 2</option>
+                    <option>Option 3</option>
+                  </select>
+                </div>
+                <div class="chi-form__item -mb--2">
+                  <label class="chi-label" for="example-ba13">Label</label>
+                  <select class="chi-select" id="example-ba13">
+                    <option value="">Select</option>
+                    <option>Option 1</option>
+                    <option>Option 2</option>
+                    <option>Option 3</option>
+                  </select>
+                </div>
+                <div class="chi-form__item -mb--2">
+                  <label class="chi-label" for="example-ba14">Label</label>
+                  <select class="chi-select" id="example-ba14">
+                    <option value="">Select</option>
+                    <option>Option 1</option>
+                    <option>Option 2</option>
+                    <option>Option 3</option>
+                  </select>
+                </div>
+              </div>
+              <div class="-d--flex -justify-content--center -pb--2">
+                <button class="chi-button -primary">Apply</button>
+                <button class="chi-button -ml--2">Cancel</button>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="chi-backdrop -closed">
+          <div class="chi-backdrop__wrapper">
+            <div class="chi-drawer -right -menu -position--absolute" id="drawer-2">
+              <div class="chi-drawer__header">
+                <span class="chi-drawer__title">Actions</span>
+                <button class="chi-button -icon -close" aria-label="Close">
+                  <div class="chi-button__content">
+                    <i class="chi-icon icon-x" aria-hidden="true"></i>
+                  </div>
+                </button>
+              </div>
+              <div class="chi-drawer__content -px--2">
+                <ul class="-no-style">
+                  <li class="-text--md">Download data</li>
+                  <div class="chi-divider"></div>
+                  <li class="-text--md">Refresh results</li>
+                  <div class="chi-divider"></div>
+                </ul>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <!-- Javascript -->
+        <script>
+          chi.drawer(document.getElementById('drawer-trigger-brbap1'));
+          chi.drawer(document.getElementById('drawer-trigger-brbap2'));
+          chi.tooltip(document.querySelectorAll('[data-tooltip]'));
+          chi.dropdown(document.getElementById('example__portal_brightspeed_save_views_bulk_actions_button_download'));
```

Comment:
Check also this id

![image](https://user-images.githubusercontent.com/86958774/198232159-03147ab4-f3e9-45b6-9a73-fbcc842210dc.png)

```suggestion
          chi.dropdown(document.getElementById('example__save_views_bulk_actions_button_download'));
```